internal/core/workers/runtime_watcher_worker/runtime_watcher_worker_test.go

```diff
@@ -138,4 +138,63 @@ func TestProcessRuntimeEvents(t *testing.T) {
 			return true
 		}, time.Second, time.Millisecond)
 	})
+
+	t.Run("updates clean room state on delete event", func(t *testing.T) {
```

Comment:
@gabrielcorado WDYT about adding a test case for when `w.roomManager.CleanRoomState` fails?

Reply:
Sure.